### PR TITLE
test: refactor utils.waitForEvents into utils.waitEvent

### PR DESCRIPTION
**Pull request by aslushnikov**
This simplifies waiting for events in tests.

### CODE REVIEW

File: test/page.spec.js [hunk 1]
```diff
@@ -380,7 +377,7 @@ module.exports.addTests = function({testRunner, expect, defaultBrowserOptions, p
     describe('Page.Events.DOMContentLoaded', function() {
       it('should fire when expected', async({page, server}) => {
         page.goto('about:blank');
-        await waitForEvents(page, 'domcontentloaded', 1);
+        await waitEvent(page, 'domcontentloaded', 1);
```
Comment (JoelEinbinder) [inline test/page.spec.js]: extra 1 here

File: test/page.spec.js [hunk 2]
```diff
@@ -340,20 +340,17 @@ module.exports.addTests = function({testRunner, expect, defaultBrowserOptions, p
...
-            console.time('calling console.time');
-            console.timeEnd('calling console.time');
-            console.trace('calling console.trace');
-            console.dir('calling console.dir');
-            console.warn('calling console.warn');
-            console.error('calling console.error');
-            console.log(Promise.resolve('should not wait until resolved!'));
-          }),
-          // Wait for 5 events to hit - console.time is not reported
-          waitForEvents(page, 'console', 5)
```
Comment (JoelEinbinder) [inline test/page.spec.js:355]: This works because console logs wait for the ack in JavaScript?

**Review by JoelEinbinder — APPROVED** [review]

File: test/page.spec.js (hunk 1 above)
Comment (aslushnikov) [reply test/page.spec.js]: Done.

File: test/page.spec.js (hunk 2 above)
Comment (aslushnikov) [reply test/page.spec.js:355]: That's because protocol is serial: console.log's will be posted before the evaluation response.

**Pull request merged**